project/src/pages/movie-page/movie-page.tsx

```diff
@@ -9,26 +9,35 @@ import Reviews from '../../components/ui/tabs/reviews';
 import {useAppDispatch, useAppSelector} from '../../hooks';
 import {AuthorizationStatus} from '../../const';
 import {getAuthorizationStatus} from '../../store/auth-slice/selectors';
-import {getFilm, getSimilarFilms} from '../../store/film-slice/selectors';
+import {getFilm, getIsLoadedFilm, getSimilarFilms} from '../../store/film-slice/selectors';
 import {useEffect} from 'react';
 import {fetchCommentsAction, fetchFilmAction, fetchSimilarFilmsAction} from '../../services/api-action';
+import ButtonFavorite from '../../components/ui/button-favorite/button-favorite';
+import Loader from '../../components/ui/util-components/loader/loader';
 
 const FILMS_COUNT = 4;
 
 function MoviePage(): JSX.Element {
   const authorizationStatus = useAppSelector(getAuthorizationStatus);
   const film = useAppSelector(getFilm);
   const similarFilms = useAppSelector(getSimilarFilms).slice(0, FILMS_COUNT);
+  const isLoading = useAppSelector(getIsLoadedFilm);
 
   const {id} = useParams();
```

Comment:
```suggestion
  const {id} = useParams<{id: string}>();
```

это поможет тебе потом не писать as string